qualification/Jenkinsfile

```diff
@@ -0,0 +1,147 @@
+/*******************************************************************************
+ * Copyright (c) 2021-2022, National Research Foundation (SARAO)
+ *
+ * Licensed under the BSD 3-Clause License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ *   https://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+/* This file tells Jenkins how to execute the qualification tests for this repo.
+ * Everything runs in docker containers, so in theory any Jenkins server should
+ * be able to parse this file, although the Jenkins server needs access to a GPU
```

Comment:
Technically it could parse this file, but in practice it's totally unportable because pytest.ini hard-codes all the information about NIC device and master controller address.